script.js

```diff
@@ -0,0 +1,85 @@
+// 1.) Go through the `items` and find all results that have `kind` of `shopping#product`. Print the count of these results. Where else is this count information stored in the search results?
```

Comment:
You can save typing product.items multiple times if you declare it into a variable like

var shopItems = product.items[0];